src/gcp/frameworks.ts

```diff
@@ -162,3 +165,11 @@ export async function createBuild(
 
   return res.body;
 }
+
+/**
+ * Ensure that Frameworks API is enabled on the project.
+ */
+export async function ensureApiEnabled(options: any): Promise<void> {
+  const projectId = needProjectId(options);
+  return await ensure(projectId, API_HOST, "frameworks", true);
```

Comment:
I could be wrong but didn't we want to ensure all of the dependent APIs as well (in case the user manually toggles any of them off?)

Reply:
Hmm yeah let me tackle that problem in the follow up PR.